rpi_backlight.py

```diff
@@ -100,7 +104,20 @@ def set_power(on: bool) -> None:
     """
 
     # 0 is on, 1 is off
-    _set_value("bl_power", int(not on))
+    if int(_get_value("tinker_mcu_bl")) == 0:
+        if on == True:
+            _set_value("tinker_mcu_bl", 255)
+    else:
+        if on == False:
+            _set_value("tinker_mcu_bl", 0)
+
+def toggle_power():
+    """Toggle the display power on or off.
+    """
```

Comment:
Put the closing `"""` on the line above please

Reply:
fixed